source/includes/mongodb-compatibility-table-c.rst

```diff
@@ -20,71 +32,80 @@
      -
 
    * - Version 1.10
-     - 
+     -
+     -
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      -
 
    * - Version 1.9
-     - 
+     -
+     -
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
    * - Version 1.8
-     - 
+     -
+     -
      -
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
    * - Version 1.7
-     - 
+     -
+     -
      -
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
    * - Version 1.6
-     - 
+     -
+     -
      -
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
    * - Version 1.5
-     - 
+     -
+     -
      -
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
    * - Version 1.3
-     - 
+     -
+     -
      -
      -
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
    * - Version 1.1.0
-     - 
+     -
```

Comment:
While you're updating this, I think you can simply drop the tailing ".0" from 1.1.0 and 1.0.0.